src/file/writer/record_writer.rs

```diff
@@ -0,0 +1,5 @@
+use super::RowGroupWriter;
+
+pub trait RecordWriter<T> {
```

Comment:
Can you move this trait into `record` module? It already has record reader there.

Reply:
Done!